docs/ios-sdk.md

```diff
@@ -203,75 +206,27 @@ liveSDK.audioSessionSettings?.options = [.mixWithOthers]
-| `maxRecoveryDelay` | `Integer`        | `90`       | Maximum delay in seconds between subsequent recovery attempts.                   |
+| Name               | Type                           | Default    | Description                                                                                                       |
+| ------------------ | ------------------------------ | ---------- | ----------------------------------------------------------------------------------------------------------------- |
+| `autoplay`         | `Boolean`                      | `false`    | Determines whether video shall play immediately after [`createPlayer`](#sdk-create-player).                       |
+| `cast`             | `Boolean`                      | `false`    | Determines whether cast button should be displayed or not on the [`Player Controls`](#player-controls).           |
+| `error`            | `Boolean`                      | `false`    | Determines whether [`Error Overlay`](#error-overlay) should be displayed or not.                                  |
+| `fit`              | [`playerFit`](#player-fitting) | `.contain` | Determines how the video will be scaled and cropped. See: [`Player Fitting`](#player-fitting)                     |
+| `fullscreen`       | `Boolean`                      | `false`    | Determines whether full screen button should be displayed or not on the [`Player Controls`](#player-controls).    |
+| `maxRecoveryDelay` | `Integer`                      | `90`       | Maximum delay in seconds between subsequent recovery attempts.                                                    |
+| `minRecoveryDelay` | `Integer`                      | `10`       | Minimum delay in seconds before starting automatic recovery.                                                      |
+| `mute`             | `Boolean`                      | `false`    | Determines whether mute button should be displayed or not on the [`Player Controls`](#player-controls).           |
+| `muted`            | `Boolean`                      | `false`    | Determines whether media should be muted or not.                                                                  |
+| `poster`           | `String`                       | `""`       | URL for an image to be shown while the video is loading.                                                          |
+| `quality`          | `Boolean`                      | `false`    | Determines whether select quality button should be displayed or not on the [`Player Controls`](#player-controls). |
+| `sources`          | `String[]`                     | `[]`       | Array of media source URLs from which the first that can be played will be selected.                              |
+| `targetLatency`    | `Integer`                      | `3`        | Target live latency in seconds. If 0 then syncing is disabled.                                                    |
 
 ### Source Protocols
```

Comment:
I think we can remove the `Source Protocols` chapter. It's not something the customers can change or use.